packages/ember/src/components/nrg/mktg/header.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@glimmer/component';
+
+export default class Header extends Component {
+  iconClasses = this.args.iconClasses || '';
+  titleClasses = this.args.titleClasses || '';
+  navClasses = this.args.navClasses || '';
```

Comment:
Also, I have a thought about this - in some places, we build the classes as an array before joining them with spaces. We do this to reduce extraneous whitespace in the DOM. You can see what I mean [here](https://github.com/knoxville-utilities-board/nrg-ui/blob/fe2e3acf6070d040c9f2512e934a27cdcc18890b/packages/ember/src/components/nrg/button.gts#L29-L37)

Reply:
So - for clarity, do you think it would be best to keep the named blocks and implement getters or use separate components that are nested together?

Reply:
I think, despite the fact that I suggested named blocks, that I would prefer yielded components. Take a look at the [`ButtonGroup`](https://github.com/knoxville-utilities-board/nrg-ui/blob/fe2e3acf6070d040c9f2512e934a27cdcc18890b/packages/ember/src/components/nrg/button-group.gts#L79-L91) - notice that it also provides a `yield` block, but instead of just placing that yield when content would go, it provides a component that users can invoke ([see example](https://github.com/knoxville-utilities-board/nrg-ui/blob/fe2e3acf6070d040c9f2512e934a27cdcc18890b/packages/ember-test-app/tests/integration/components/nrg/button-group-test.ts#L86-L92)).